rllib/tuned_examples/ppo/multi_agent_pendulum_ppo.py

```diff
@@ -8,17 +8,21 @@
 config = (
     PPOConfig()
     .environment("multi_agent_pendulum")
-    .rollouts(num_envs_per_worker=10, batch_mode="complete_episodes")
+    .rollouts(
+        num_envs_per_worker=20,
+        observation_filter="MeanStdFilter",
+        num_rollout_workers=0,
+    )
     .training(
-        train_batch_size=2048,
+        train_batch_size=512,
         lambda_=0.1,
-        grad_clip=0.95,
+        gamma=0.95,
         lr=0.0003,
         sgd_minibatch_size=64,
-        num_sgd_iter=10,
-        model={"fcnet_hiddens": [128, 128], "fcnet_activation": "relu"},
+        model={"fcnet_activation": "relu"},
```

Comment:
The above changes reflect the HPs that we have in our single agent pendulum ppo yaml. ("pendulum-ppo.yaml")
Under the RL Modules stack, this brings a significant boost in learning behaviour.